README.md

```diff
@@ -189,3 +189,42 @@ In dark mode!
 
 ## :page_with_curl: License
 This project is licensed under the MIT License - see the [LICENSE](LICENSE) file for details.
+
+
+## Instructions to run
```

Comment:
The instructions are not necessary as that is already present in details in the README.

Reply:
the changes are applied